src/lib/command/util/presentation-table.ts

```diff
@@ -81,46 +89,3 @@ export function buildTableOutput(tableGenerator: TableGenerator, presentation: P
 		`${automationConditions}\n\n` +
 		automationActions
 }
-
-export default class PresentationCommand extends APICommand<typeof PresentationCommand.flags> {
-	static description = 'query device presentation by vid' +
-		this.apiDocsURL('getDevicePresentation')
-
-	static flags = {
-		...APICommand.flags,
-		...outputItem.flags,
-	}
-
-	static args = [
-		{
-			name: 'presentationId',
-			description: 'system generated identifier that corresponds to a device presentation',
-			required: true,
-		},
-		{
-			name: 'manufacturerName',
-			description: 'manufacturer name. Defaults to SmartThingsCommunity',
-			required: false,
-		},
-	]
-
-	static examples = [
-		'$ smartthings presentation fd4adb7f-4a23-4134-9b39-05ed889a03cf',
-		'$ smartthings presentation 4ea31e30-2aba-41c7-a3ec-8f97423d565a SmartThings',
-		'$ smartthings presentation fd4adb7f-4a23-4134-9b39-05ed889a03cf --language=ko',
-		'$ smartthings presentation fd4adb7f-4a23-4134-9b39-05ed889a03cf --language=NONE',
-		'',
-		'Specifying only the presentationId defaults to the "SmartThingsCommunity" manufacturer',
```

Comment:
Does this info about specifying only the ID and the defaulting of `SmartThingsCommunity` need to be added to the new version?  There is an example that is ID only that mentions `SmartThingsCommunity` but maybe not as explicitly as it does here.